### ISSUE TITLE
3.2.1 NSP panel stopped working

### ISSUE THREAD

**Issue by PowerADE1974** [Blackymas/NSPanel_HA_Blueprint#564]
Hello 

I updated to the latest version for the blueprint and tft file. I have looked at all previous issues but cannot fix my panel.

It currently shows "System data ERROR" in sideways orientation on EU panel. The physical buttons still work (left controlling a light entity, right a the physical relay 2). 
I have checked my NSPANEL name is correct (never changed from when it was working at 3.1)

![image](https://user-images.githubusercontent.com/109544599/228570150-7f2b77f4-522b-4b1b-a35b-d972d9e6173b.png)

The ESPhome device has a strange readback though
![image](https://user-images.githubusercontent.com/109544599/228575655-a6ffacb5-a5d1-4a3b-b699-04c7d5eb197b.png)

I also see this in the logs
15:05:29][E][uart:015]: Reading from UART timed out at byte 0!
[15:05:29][W][nextion:078]: Nextion is not connected! 

It is connected via WIFI and shows correctly
[15:05:21][C][wifi:367]:   Hostname: 'nspkitchen'
[15:05:21][C][wifi:369]:   Signal strength: -48 dB ▂▄▆█
[15:05:21][C][wifi:373]:   Channel: 11

Seems like its the display tft/nextion that is the issue.
Any advice is appreciated.

Best Regards
Adrian

**Comment by pkleingu** [comment]
if you reboot te system you see this?
![PXL_20230329_121155108](https://user-images.githubusercontent.com/71530042/228631092-26a605e6-db40-4721-a9e3-0e26ba073253.jpg)

**Comment by PowerADE1974** [comment]
Also if I update the TFT display though the configuaration is gets stuck at 88%

**Comment by PowerADE1974** [comment]
Thanks for the solution but this wont work for me. I have no soldering facilities. Sent from my iPhoneOn 29 Mar 2023, at 21:34, adrian priestley ***@***.***> wrote:﻿No. I see this <image0.jpeg>Sent from my iPhoneOn 29 Mar 2023, at 19:17, pkleingu ***@***.***> wrote:
if you reboot te system you see this?

**Comment by hkr086** [comment]
Following this as I have the same issue. I installed the blank TFT and the screen turned white. No I tried to re upload the TFT but it takes ages.

**Comment by PowerADE1974** [comment]
UPDATE.

I have started all over again with a clean install. Still does not work

I can change the function of the 2 physical buttons via the blueprint but the display still says
"System data ERROR"

I get this in the ESP log after the install though
[22:26:58][W][nextion:341]: Nextion reported page ID invalid!

I tried flashing with nspanel_blank.tft using the ESPHome: nspkitchen_upload_tft_url service and that does nothing at all except flashes a blank screen for a fraction of a second before returning to "System data ERROR"

I am stuck, please please help.

**Comment by hkr086** [comment]
My resolution to this was to install the TFT file from another webserver.in my case I used my nasdrive http functionality. I installed the blank TFT first and then the nspanel_eu. But used my nasdrive http function to upload from. 
Not sure why from the www folder in home assistant it did not work. I have 4 ns panel and 3 of them (white model) worked. But the Gray model did not.

I saw this somewhere in here so it was not my idea but it worked. And thank you to the guy who wrote it as I did 2 days trying with no luck. Hope it helps you as well.
